### ISSUE TITLE
Split Images into Training set and Test set

### ISSUE THREAD

**Issue by Eugeniosales** [deeplearningunb/Skincare#12]
This procedure aims to set the data apart which is going to be used to train the model and check the accuracy afterwards.

To be accepted, this procedure must contain:
- [x] A proportion of 20% for Test set and 80% for Training Set
